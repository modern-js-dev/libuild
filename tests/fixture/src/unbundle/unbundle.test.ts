import { expect, getLibuilderTest } from '@modern-js/libuild-test-toolkit';

describe('fixture:unbundle', () => {
  it('unbundle', async () => {
    const bundler = await getLibuilderTest({
      root: __dirname,
      input: ['src/*'],
      bundle: false,
    });
    await bundler.build();
    const jsOutput = bundler.getJSOutput();
    const values = Object.values(jsOutput);
    expect(values.length === 2).to.be.true;
    bundler.expectJSOutputMatchSnapshot();
  });
});
