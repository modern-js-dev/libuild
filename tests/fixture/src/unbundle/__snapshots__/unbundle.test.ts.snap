// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixture:unbundle unbundle 1`] = `
"import { a } from \\"./a\\";
console.log(a());
"
`;

exports[`fixture:unbundle unbundle 2`] = `
"const a = () => {
  return 1;
};
export {
  a
};
"
`;
